pkgs/by-name/ya/yao-pkg/package.nix

```
@@ -0,0 +1,106 @@
+{
+  autoPatchelfHook,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  lib,
+  makeWrapper,
+  nodejs,
+  stdenv,
+  yarn,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pkg";
+  version = "6.1.1";
+
+  src = fetchFromGitHub {
+    owner = "yao-pkg";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-AQ+PVIWYgrflDsauxCfmvo40imlTAOW3vXhMtN+eKq4=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${src}/yarn.lock";
+    hash = "sha256-7jIrhIuuHSkfQCnjifgYCEIHi169np78NLhuw6iZRyU=";
+  };
+
+  yarnBuildFlags = [
+    "--frozen-lockfile"
+    "--ignore-platform"
+    "--ignore-scripts"
+    "--no-progress"
+    "--offline"
+    "--prefer-offline"
+    "--production"
+    "--yarn-offline-mirror"
+  ];
+
+  nativeBuildInputs = [
+    fixup-yarn-lock
+    makeWrapper
+    nodejs
+    yarn
+    yarnBuildHook
+    yarnConfigHook
+    yarnInstallHook
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
+  autoPatchelfIgnoreMissingDeps = [ "*" ];
+
+  configurePhase = # bash
```

Even though I understand that this is probably an editor syntax highlight thing, please don't include this in Nixpkgs
```suggestion
  configurePhase =
```

---

Have def removed these, thanks for that